docs-src/docs/usingwidgets.md

```diff
@@ -0,0 +1,163 @@
+# Using Widgets
+
+[Widgets](widgetoverview.md) are how you create your GUI. They are the things which appear on the GUI, everything from the app itself to text boxes, buttons and pictures.
+
+At the start of every guizero program, choose the widgets you need from the guizero library and import them e.g.
+
+```python
+from guizero import App, Text, PushButton, Slider
+```
+
+You only need to import each widget once, and then you can use it in your program as many times as you like.
+
+### The App widget
+
+Every guizero app will need an [App](app.md) widget. This is the main window.
+
+At the end of every guizero program you must tell the program to `display` the app you have just created.
+
+```python
+from guizero import App
+
+app = App(title="My guizero app")
+
+app.display()
+```
+
+### Adding widgets
+
+All the text boxes, buttons, sliders and even plain old pieces of text which appear on your GUI are widgets.
+
+**All widgets** go between the line of code to create the `App` and the `app.display()` line.
+
+```python
+from guizero import App, Text, TextBox
+
+app = App(title="Whats your name?")
+
+# create widgets here
+message = Text(app, text="Enter your name")
+name = TextBox(app)
+
+app.display()
+```
+
+Let’s look at the [Text](text.md) widget code in a bit more detail:
+
+```python
+message = Text(app, text="Enter your name"))
+```
+
+- `message =` - The `Text` object has a name, just like any variable
+- `Text` - an *object* which creates a piece of text on the screen
+- `app` – This tells the `Text` where it will live. Most of the time your widgets will live directly inside the app.
+- `text="Enter your name"` - The text to display
+
+![an app showing the text 'Enter your name` with a text box below where a name can be entered](images/whats-your-name.png)
```

Comment:
I don't know whether it's just the github preview but this image looks broken too

Reply:
I think it must have been a problem with the preview. The built docs look fine. The preview looks ok now too.